src/Revit/RevitHyparTools/RevitExtensions.cs

```diff
@@ -1,11 +1,75 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using Autodesk.Revit.DB;
 using Elements.Geometry;
 
-namespace Hypar.Revit 
+namespace Hypar.Revit
 {
-    public static class RevitExtensions {
-        public static Vector3 ToVector3(this XYZ xyz) {
+    public static class RevitExtensions
+    {
+        public static Vector3 ToVector3(this XYZ xyz)
+        {
             return new Vector3(xyz.X, xyz.Y, xyz.Z);
-        }       
+        }
+        public static Polygon ToPolygon(this CurveLoop curveLoop)
```

Comment:
Add one line of whitespace between methods.

Reply:
done